CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing to ContextualLogger
```

Comment:
I think this should be `Contributing to DeclareSchema`. Or maybe just make it `Contributing` so it's ready for future copy/paste?

Reply:
Thanks for catching this, fixed 1e67bb195e516981f55287cd7d5c9f74b9ef586e